### PR TITLE
BE 6 - card (add appointment)

### PR DESCRIPTION
**Pull request by KuzminAanton**

### CODE REVIEW

File: src/modules/controllers/appointment.controller.js [hunk 1]
```diff
@@ -1 +1,44 @@
...
+const jwt = require('jsonwebtoken');
+const Users = require('../../db/models/users/index');
 const Appointment = require('../../db/models/appointment/index');
+const Doctors = require('../../db/models/doctors/index');
+
+module.exports.addAppointments = async (req, res) => {
+  const {
+    patientName, doctorName, date, complaints,
+  } = req.body;
+  if (req.body) {
```
Comment (usatovaDasha) [inline src/modules/controllers/appointment.controller.js:11]: it is object, object - true

File: src/modules/controllers/appointment.controller.js [hunk 2]
```diff
@@ -1 +1,44 @@
...
+    const appointmentNew = new Appointment({
+      patientName,
+      doctorName,
+      date,
+      complaints,
```
Comment (usatovaDasha) [inline src/modules/controllers/appointment.controller.js:16]: no need userId?

File: src/modules/controllers/appointment.controller.js [hunk 3]
```diff
@@ -1 +1,44 @@
...
+    });
+    await appointmentNew.save().then(() => {
```
Comment (usatovaDasha) [inline src/modules/controllers/appointment.controller.js:18]: no need async/await

File: src/modules/controllers/appointment.controller.js [hunk 4]
```diff
@@ -1 +1,44 @@
...
+          data: result,
+        });
+      });
+    });
+  } else {
+    res.status(422).send('Value is not correct');
+  }
+};
+
+module.exports.getDoctors = async (req, res) => {
```
Comment (usatovaDasha) [inline src/modules/controllers/appointment.controller.js:30]: remove async

File: src/modules/controllers/appointment.controller.js [hunk 5]
```diff
@@ -1 +1,44 @@
...
+  Doctors.find().then((result) => {
+    res.send({
+      data: result,
+    });
+  });
+};
+
+module.exports.getAppointments = async (req, res) => {
```
Comment (usatovaDasha) [inline src/modules/controllers/appointment.controller.js:38]: remove async

File: src/modules/controllers/appointment.controller.js (hunk 4 above)
Comment (KuzminAanton) [reply src/modules/controllers/appointment.controller.js:30]: fixed

**Pull request merged**